x-pack/plugins/stack_alerts/server/rule_types/index_threshold/rule_type.ts

```diff
@@ -167,6 +173,18 @@ export function getRuleType(
     }
   );
 
+  const alerts: IRuleTypeAlerts<StackAlert> = {
```

Comment:
nit: could we move this `alerts` definition to a shared location so all stack rule types can use it instead of redefining inside each rule type file?

Reply:
Good idea, moved all of them to constants.